samples/contrib/aws-samples/simple_train_pipeline/README.md

```diff
@@ -0,0 +1,60 @@
+# Simple pipeline with only train component
+
+An example pipeline with only train component.
+
+# Prerequisites 
+1. Install kubeflow on an EKS cluster in AWS. https://www.kubeflow.org/docs/aws/deploy/install-kubeflow/
+2. Get and store data in S3 buckets. You can get sample data using this code  
+   `aws s3 cp s3://m-kfp-mnist/mnist_kmeans_example/data s3://<your_bucket_name>/mnist_kmeans_example/data`
```

Comment:
Is this `m-kfp-mnist` bucket made public? Does the SageMaker team own it?

Reply:
nice catch 
let me check it out